generators/search-map.js

```diff
@@ -4,7 +4,8 @@ var fs = require('fs'),
 	Q = require('q'),
 	writeFile = Q.denodeify(fs.writeFile),
 	mkdirs = Q.denodeify(require("fs-extra").mkdirs),
-	sanitizeHtml = require('sanitize-html');
+	markdown = require('github-flavored-markdown'),
```

Comment:
`bit-docs` uses something called `stmd.js`: https://github.com/bit-docs/bit-docs-generate-html/blob/master/stmd.js. Although it is on NPM, the main project doesn't have that source anymore. Some forks of it do though, which seems kinda odd. Would you prefer I add the file directly to the project instead of using a different parser?